packages/patterns/src/keys/checkKey.js

```diff
@@ -432,6 +432,23 @@ export const getCopyMapValues = m => {
 };
 harden(getCopyMapValues);
 
+/**
+ * Returns an array of a CopyMap's entries in storage order.
+ *
+ * @template {Key} K
+ * @template {Passable} V
+ * @param {CopyMap<K,V>} m
+ * @returns {Array<[K,V]>}
+ */
+export const getCopyMapEntryArray = m => {
```

Comment:
No objection. But is this really better than

```suggestion
export const getCopyMapEntryArray = m =>
  harden([...getCopyMapEntries(m)]);
```
?

Reply:
Given that the change was motivated in part by efficiency, I thought it best to avoid immediately-collapsed iterators (and also `getCopyMapEntries` just feels less fundamental, since it returns a Far object).